pkg/builder/local_build_executor.go

```diff
@@ -104,6 +106,8 @@ func (be *localBuildExecutor) uploadDirectory(ctx context.Context, outputDirecto
 				Name:   name,
 				Target: target,
 			})
+		case filesystem.FileTypeOther:
```

Comment:
It's annoying that Go doesn't have true native enum types and that `switch` can be tagged to require that all cases are handled.

Aaanyway: I propose that we replace all of this by just:
```go
default:
    // A UNIX socket, FIFO or device node. These files cannot be represented in
    // a Directory message. Simply skip them. Returning an error here would
    // make the overall user experience worse.
    continue
```

Reply:
Or wait. We don't even need the `default: continue`. There is nothing after this switch statement, so there's no need to `continue`. In that case: just place a similar comment above the switch statement.